src/pages/events/commands.md

```diff
@@ -119,12 +121,25 @@ You can also subscribe to a plugin event if it was registered in the `app/etc/co
 
 `--force`, `-f` Forces subscription to the event, even if it hasn't been defined locally.
 
+`--parent <event code>` Specifies the name of the event to use as the basis of an event rule. If you specify this option, you must also specify the `--rules` option.
```

Comment:
This is very small, but this could be labeled as `--parent=<event code>` similar to how the rules option below is labeled